pom.xml

```diff
@@ -122,6 +122,8 @@
         <maven-scm-plugin.version>1.11.2</maven-scm-plugin.version>
         <org.eclipse.jgit.version>5.4.0.201906121030-r</org.eclipse.jgit.version>
         <jacoco.version>0.8.4</jacoco.version>
+		<sonar-jacoco-listeners.version>5.14.0.18788</sonar-jacoco-listeners.version>
```

Comment:
Wrong indentation.

Reply:
Ohh, my eclipse's xml editor was using tab instead of spaces. I'll fix them now.